sre-operational-fabric.md

```diff
@@ -0,0 +1,119 @@
+# (Remote) SRE - Rackspace Private Cloud
+
+## Role
+
+Site Reliability Engineering (SRE) is an engineering discipline that combines
+software and systems engineering to build and run large-scale, massively
+distributed, fault-tolerant systems. SRE ensures that Rackspace's managed
+service offerings & customer deployments have reliability and uptime
+appropriate to users' needs and a fast rate of improvement while monitoring and
+validating capacity and performance. Act as a Technical Lead for a portfolio of
+managed service deployments. Focus on reliability, scalability and the
+development of automation to manage a set of repetitive tasks at scale.
+
+### Knowledge, Skills, Ability
+
+- Deep Experience in one or more of: C, C++, Java, Perl, Python, Bash or Go.
+- Deep experience working with Unix/Linux systems from kernel to shell and beyond, with experience working with system libraries, file systems, and client-server protocols.
+- Networking: e.g. TCP/IP, UDP, ICMP, etc., MAC addresses, IP packets, DNS, SDN, OSI layers, and load balancing.
+- Strong expertise in designing, analyzing and troubleshooting large-scale distributed systems.
+- In depth knowledge of operating systems.
```

Comment:
s/In depth/Deep

Someone can talk to you about knowledge of operating systems in depth, but they don't have "in depth" knowledge. (I'm forgetting the specific grammatical term here to explain this, but you can substitute "thoroughly" for "in depth" and see what I mean.)

Reply:
updated